bin/extract-job-output

```diff
@@ -61,6 +61,9 @@ for image_path in paths:
             # print "  - creating directory"
             os.makedirs(local_dir)
             # print "  - extracting"
-        g.download(full_image_path, os.path.join(output_dir, image_path))
+        try:
```

Comment:
This just hides errors. (Was this an intentional change or did it sneak in from a debugging session?)